pkg/api/handler.go

```diff
@@ -141,6 +141,11 @@ func (h *handlerState) Report(ctx context.Context, tracker attribute.Tracker, re
 
 // Quota performs 'quota' function corresponding to the mixer api.
 func (h *handlerState) Quota(ctx context.Context, tracker attribute.Tracker, request *mixerpb.QuotaRequest, response *mixerpb.QuotaResponse) {
+	if glog.V(2) {
+		glog.Infof("Quota [%x]", request.RequestIndex)
+		defer func() { glog.Infof("Quota [%x] <-- %s", request.RequestIndex, response) }()
```

Comment:
nit: deferring in logging might cause hard-to-debug memory issues if there're blocking calls inside the body